src/rtde/rtde_client.cpp

```diff
@@ -37,8 +37,8 @@ namespace rtde_interface
 RTDEClient::RTDEClient(std::string robot_ip, comm::INotifier& notifier, const std::string& output_recipe_file,
                        const std::string& input_recipe_file, double target_frequency)
   : stream_(robot_ip, UR_RTDE_PORT)
-  , output_recipe_(readRecipe(output_recipe_file))
-  , input_recipe_(readRecipe(input_recipe_file))
+  , output_recipe_(readRecipe(output_recipe_file, RecipeType::OUTPUT_RECIPE))
```

Comment:
```suggestion
  , output_recipe_(ensureTimestampIsPresent(readRecipe(output_recipe_file)))
```

- [ ] remove enum as well